### PR TITLE
Document ACL support in the Windows Network Drive app

### LINKED ISSUE #768: ACL support for WND
## What Needs to be Documented?

https://github.com/owncloud/windows_network_drive/pull/234

## Why Should This Change Be Made? (Optional)

New content planned for the next WND release. It will require documentation because there are several things to notice about the implementation.

## What Type Of Content Change Is This? (Optional)
- [x] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [ ] Bug Fix to Existing Content

## Which Manual Does This Relate To? (Optional)
- [x] Admin Manual
- [ ] Developer Manual
- [ ] User Manual
- [ ] Android
- [ ] iOS
- [ ] Branded Clients
- [ ] Desktop Client
- [ ] Other

@settermjd there is some information in the PR, but there could be more information that we might need to add. This should be for the next WND (not sure when). I can provide additional information about the implementation, so feel free to ping me.
Also pinging to @felixboehm and @PVince81 for them to be aware

### PR DESCRIPTION
**Pull request by settermjd**
This relates to https://github.com/owncloud/windows_network_drive/pull/234, https://github.com/owncloud/windows_network_drive/issues/243, and fixes #768.

### CODE REVIEW

File: modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc [hunk 1]
```diff
@@ -161,6 +163,39 @@ Your changes are saved automatically.
...
+===== The ownCloud LDAP Permission Manager
+
+To use ACLs from an LDAP server, whether that is a Windows Active Directory server or a Samba server, use the ownCloud LDAP Permission Manager.
+To use this provider requires two, key, things:
+
+- An Active Directory server containing standard user and group information that can be used by {ldap-app-url}[ownCloud's LDAP app].
+- ownCloud's LDAP app to be properly configured to retrieve user and group information from the Active Directory server.
+
+IMPORTANT: The ownCloud LDAP apps must configure the `sAMAccountName` to be the ownCloud server's username.
+
```
Comment (jvillafanez) [inline modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc]: Adding some information:
The ownCloud LDAP Permission Manager will evaluate the ACL in the file along with the file attributes in order to determine the permissions. Both will be evaluated

File: modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc [hunk 2]
```diff
@@ -161,6 +163,39 @@ Your changes are saved automatically.
...
+===== The Null Permission Manager
+
+This is the default permission manager, used if no other ACL manager is specified.
+If you want to retain ownCloud’s current behaviour, then use this permission manager. 
+When in effect, the Windows Network Drive app will use a file’s attributes (e.g., read-only, and hidden), to determine how the user can interact with the file.
+There are no usage restrictions.
+
+===== The ownCloud LDAP Permission Manager
+
+To use ACLs from an LDAP server, whether that is a Windows Active Directory server or a Samba server, use the ownCloud LDAP Permission Manager.
```
Comment (jvillafanez) [inline modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc]: The ACLs come from the windows / samba server.
The requirement is that both the windows / samba server and also ownCloud's user_ldap app are connected to the same AD, so ownCloud can retrieve the same user and group information that the SMB server is using. Other setups are unlikely to work properly.

File: modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc [hunk 3]
```diff
@@ -161,6 +163,39 @@ Your changes are saved automatically.
...
+To use this provider requires two, key, things:
+
+- An Active Directory server containing standard user and group information that can be used by {ldap-app-url}[ownCloud's LDAP app].
```
Comment (jvillafanez) [inline modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc]: As said, remark that the AD must be the same one being used by the SMB server (windows / samba). It won't work properly if it's connected to a different AD.

**Comment by jvillafanez** [comment]
> Admin trigger a files:scan manually after changing the configuration

This should be also documented. When you change the permission manager in the UI, you should run the files:scan to update the permissions correctly, otherwise the permissions shown by ownCloud might be different (permissions are only auto-updated if there has been a change in the files)

In addition, include the names of the permissions managers: "nullPermissionManager" and "ocLdapPermissionManager"; otherwise I don't think the admins will know how to fill the textbox. The names must match exactly.

**Comment by settermjd** [comment]
All feedback has been addressed. Can you please review again, @jvillafanez?

File: modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc (hunk 2 above)
Comment (jvillafanez) [reply modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc]: I think this sentence is confusing and we should remove it.

Maybe the one below is clearer.
> The ownCloud LDAP Permission Manager evaluates ACLs in files along with file attributes to determine the permissions. In order to evaluate the ACLs, we'll need access to the user and group membership information of the target windows / samba server, so we'll use the ownCloud's user_ldap app for this.

Comment (settermjd) [reply modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc]: Looks good to me.

**Pull request merged**